bridges/scalajs-1/src/main/scala/bloop/scalajs/jsenv/NodeJsEnv.scala

```diff
@@ -25,6 +24,11 @@ import org.scalajs.jsenv.JSUtils.escapeJS
 
 import scala.collection.JavaConverters._
 import scala.concurrent.{Future, Promise}
+import scala.concurrent.ExecutionContext.Implicits.global
```

Comment:
```suggestion
```

Let's no use global

Reply:
It's used [here](https://github.com/scalacenter/bloop/pull/1531/files#diff-9e00bb9ceb3e05cac1e8ff00ead15d96710162b0d36489715128d67a0d13d80aR296). I moved that import close to its usage.